basics/serviceEndpoints.ts

```diff
@@ -0,0 +1,59 @@
+import { ConfigService, DidHelpers } from '@kiltprotocol/sdk-js'
+import { DidDocument, KiltAddress, SignerInterface } from '@kiltprotocol/types'
+
+const serviceId = '#example-service-id'
+
+export async function addServiceEndpoints(
+  didDocument: DidDocument,
+  signers: SignerInterface[],
+  submitter: SignerInterface<'Ed25519', KiltAddress>
+): Promise<void> {
+  const api = ConfigService.get('api')
+
+  const transactionHandler = DidHelpers.addService({
+    api,
+    didDocument,
+    signers: signers,
+    submitter: submitter,
+    service: {
+      id: serviceId,
+      type: ['example-type'],
+      serviceEndpoint: ['https://example.com/endpoint'],
+    },
+  })
+
+  const result = await transactionHandler.submit()
+
+  // Check if the result is rejected or confirmed
+  if (result.asRejected) {
```

Comment:
```suggestion
  // Check if the result is rejected or confirmed
  if (result.isRejected) {
```
this would throw already if it is not rejected